tests/code_layout/test_banned_keywords.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
```

Comment:
IIRC it was recommended to use `/usr/bin/sh` to avoid "bashism"

Reply:
Actually every other .sh file uses `/usr/bin/env bash`, including all those from the code_layout tests

Reply:
`sh` lives in `/bin` anyway.   And a bashism is (sloppy) use of a bash extension in script meant to be executed with `/bin/sh` - which will fail if `/bin/sh` is not a bash - which is used to be on Linux for long.   So everything fine - bash should be intentional as `test_banned_keywords.sh` uses bash extensions (`declare`, arrays…)